lib/matplotlib/tests/test_lines.py

```diff
@@ -7,7 +7,7 @@
 import six
 
 import nose
-from nose.tools import assert_true, assert_raises
+from nose.tools import assert_true, assert_raises, assert_raises_regexp
```

Comment:
Not sure that it's relevant but IPython dropped support some time ago with version 2.0

More to the point I guess we can maintain support for it as long as we can test it on Travis and the work arounds are limited to minor issues like this.